src/controller/python/chip-device-ctrl.py

```diff
@@ -816,6 +816,51 @@ def zcl_command(
     except Exception as e:
         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
+def zcl_read_attribute(
+        cluster: str,
+        attribute: str,
+        node_id: int,
+        endpoint_id: Optional[int] = 0,
+        group_id: Optional[int] = 0):
+    """Generic RPC call to read ZCL cluster attributes
+    zclread command format: zclread <Cluster> <Attribute> <NodeId> <EndpointId> <GroupId> [arguments]
+    
+    Args:
+        cluster (str): Name of cluster
+        command (str): Command to be run for the mentioned cluster
```

Comment:
There is no `command` arg, should this be `attribute`? please fix.

Reply:
Done